### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by WilliamMS-git**
Intentionally changed URL, because I changed my username

### CODE REVIEW

**Comment by WilliamMS-git** [comment]
Intentionally changed URL, because I changed my username

**Review by per1234 — COMMENTED** [review]
Hello @WilliamMS-git. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/02/08 16:50:08 Loaded 6630 libraries from DB
Changing URL of library ADXL372 from https://github.com/Fourier-git/ADXL372.git to https://github.com/WilliamMS-git/ADXL372.git
Success!
```

**Pull request merged**
